src/screens/FeedScreen/ContentPoll.js

```diff
@@ -309,15 +310,15 @@ const stylesComponent = (pollLength) =>
       alignSelf: 'center'
     },
     seeresultstext: {
-      color: COLORS.holyTosca,
+      color: colors.holytosca,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property has been updated from `COLORS.holyTosca` to `colors.holytosca`. Make sure that `colors.holytosca` is defined and is the intended color.
```diff
- color: COLORS.holyTosca
+ color: colors.holytosca
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->